go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/evanphx/json-patch v4.11.0+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/go-bindata/go-bindata v3.1.1+incompatible
+	github.com/go-openapi/spec v0.20.3
```

Comment:
we don't want to add this back in... is this really required by the new govmomi version?

Reply:
agreed. will take a look today to see from where this is coming and see If we can eliminate it upstream